template/package.json

```diff
@@ -4,19 +4,30 @@
   "description": "{{ description }}",
   "author": "{{ author }}",
   "license": "{{ license }}",
-  {{#if_eq license "UNLICENSED"}}
   "private": true,
-  {{/if_eq}}
   "scripts": {
     "build": "roc build",
     "dev": "roc dev",
     "start": "roc start"
   },
   "dependencies": {
-    "roc-package-web-app-react": "^1.0.0-beta.3"
+    "babel-eslint": "^7.1.0",
+    "eslint": "^3.6.0",
```

Comment:
👋 
@dlmr has a local working thing here, which I will introduce once that has been pushed and published ✅ For now I added a npmscript on the branch.